build/stf-run-ci/tasks/main.yml

```diff
@@ -75,6 +75,11 @@
     when:
     - __loki_skip_tls_verify | bool
 
+  - name: Replace loki-operator golang base image
+    replace:
+      path: "{{ playbook_dir }}/working/loki-operator/Dockerfile"
+      regexp: "golang:1.16"
+      replace: "quay.io/jwysogla/golang:latest"
```

Comment:
We'll want to replace this path with a default value in `defaults/main.yaml` and then make it so you can override it via `ansible-playbook -e ...`